status-im/nim-status-client
# fix: bump status-go to bug/#2027

Creating this PR so we can test a build with the updated sqlite paths.

Test procedure is as follows:
1. Install an older version
2. Create an account
3. Make sure db files are created "app-xxxxx.."
4. Install new version
5. Login, make sure login works
6. Look at db files, they should be a different format "xxxx.db"
